sources/packages/api/src/services/student-scholastic-standings/student-scholastic-standings.model.ts

```diff
@@ -0,0 +1,10 @@
+export interface ScholasticStanding {
+  dateOfChange?: string;
```

Comment:
Is dateOfChange also optional? just to clarify

Reply:
yes, its